code/game/objects/items/storage/backpack.dm

```diff
@@ -889,6 +894,20 @@ GLOBAL_LIST_EMPTY_TYPED(radio_packs, /obj/item/storage/backpack/marine/satchel/r
 	fuel_type = "utnapthal"
 	has_gamemode_skin = TRUE
 
+/obj/item/storage/backpack/marine/engineerpack/flamethrower/afterattack(obj/O as obj, mob/user as mob, proximity)
```

Comment:
remove the `as mobs` and stuff here, and make the variable not O

Reply:
i am not sure what you mean exactly by "remove the as mobs and stuff here" but i will look into it.
will probably look into other afterattack example....
for the variable O .... i will look into it but if this variable is all over the place....
please i hope not.... argggggggg